_tour/multiple-parameter-lists.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: tour
+title: Multiple Parameter Lists
+
+discourse: true
+
+partof: scala-tour
+
+num: 10
+next-page: case-classes
+previous-page: nested-functions
+
+redirect_from: "/tutorials/tour/multiple-parameter-lists.html"
+---
+
+Methods may define multiple parameter lists. When a method is called with a fewer number of parameter lists, then this will yield a function taking the missing parameter lists as its arguments.
```

Comment:
"This is formally known as currying." and then maybe a link to the currying wikipedia page? https://en.wikipedia.org/wiki/Currying

Reply:
I agree, but I would also suggest changing the title to "Multiple Parameter Lists (Currying)". That would make it easy for people to recognise sections according to specific names they have heard of, with regards to functional programming. One often hears about the formal names for features of the language.

In general, I think that titles should contain a pedagogical title part, such as "multiple parameter lists" and a pat containint some sort of reference to the more formal name.

Reply:
@tfinneid , I am not sure how much value there is in adding the formal name to the title as well. Although I do not see a downside to it either (other than rather long section names). I would like @jvican 's opinion on this before I make my edits.

Reply:
I see value in having the technical term in the title just for those that are already familiar with the term and want to see what the Scala tour says about it, or how Scala allows to encode currying in the language. :smile: